ci-operator/config/oadp-qe/oadp-e2e-qe/oadp-qe-oadp-e2e-qe-release-v1.3.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  to: master
```

Comment:
Shouldn't this be `release-v1.3` because of the branch you are using?

Reply:
usually we create a branch release-vx.y when we start working on the a new major release.
SO for now it will be master. 
When we start with 1.4 then i need to cut the release-v1.3 branch

Reply:
done